### ISSUE TITLE
[Webcompat reporting tool][Accessibility - VoiceOver] - "Thank you for your feedback" is not announced after submitting the report

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#35325]
**Steps to Reproduce**

1. Fresh install Firefox.
2. Access a webpage and tap on the 3-dot menu.
3. Expand the menu and tap on the "Report Broken Site" button.
4. Enable VoiceOver and fill in the report.
5. Submit the report.
6. Listen for the confirmation message.

**Expected Behavior**

VoiceOver announces "Thank you for your feedback" after the report is successfully submitted.

**Actual Behavior**

VoiceOver does not announce "Thank you for your feedback" after submitting the report.

**Device & Build Information**

- Device: iPhone 16 (iOS 26.6)
- Build version: v9000 (75395)
- First seen version: N/A

**Notes: Manually, after I tapped on the toast, it was read correctly.**

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-16510)
